README.md

```diff
@@ -12,55 +12,54 @@ Currently supports the following platform:
 - [x] OSX
 - [x] Linux
 
+The project is another runtime for your [Node.js][] packages, while ShadowNode is designed to be used on memory limited devices. It's inspired and forked from the awesome project [Samsung/iotjs][].
```

Comment:
Should "packages" be corrected to "applications"?

Reply:
Statement "package" might including "application" as well, whilst "application" limits the statement to a narrower definition of programs could be ran.